### PR TITLE
Fix bug #220: disable handleTouch doesn't work

### PR DESCRIPTION
**Pull request by amhofny**
## Description

Disabling handleTouch is not working, if you set to false, the default value true will always be used, and the same goes for injectBaseStyles.

- Fix issue #220   

## Checklist

Please use the checklist that is most closely related to your PR _(you only need to use one checklist, and you can skip items that aren't applicable or don't make sense)_:

### Bug Fix

 All existing unit tests are still passing (if applicable)
- [x] Add new passing unit tests to cover the code introduced by your PR
- [x] The PR title is in the [conventional commit](#conventional-commit-spec) format: e.g. `fix(<area>): fixed bug #issue-number`

## Steps to Test

Just set handleTouch to false and check the result, a more appropriate way is to edit the test case "accepts custom settings", like below to include "handleTouch" and "injectBaseStyles", and then run tests:
```js
it("accepts custom settings", () => {
      const anchor = document.querySelector(".test-anchor");
      const drift = new Drift(anchor, { inlineOffsetX: 12, handleTouch: false,
                                        injectBaseStyles: false });

      const expectedConfig = defaultDriftConfig();
      expectedConfig.inlineOffsetX = 12;
      expectedConfig.handleTouch = false;
      expectedConfig.injectBaseStyles = false;

      expect(drift.settings).toEqual(expectedConfig);
    });
```

Related issue: #220

### CODE REVIEW

**Comment by sherwinski** [comment]
Hey @amhofny, thanks for the PR! I just have a few changes I'd like to see before we go ahead and merge this. 
For one thing, we try not to check in `dist/*` files into branches as it creates a lot of merge conflict-headaches down the line. It would be great if you could remove those from this PR.

File: src/js/Drift.js [hunk 1]
```diff
@@ -44,7 +44,8 @@ export default class Drift {
     // if `false`, it will switch to inline when `windowWidth <= inlinePane`
     const inlinePane = options["inlinePane"] || 375;
     // If `true`, touch events will trigger the zoom, like mouse events.
-    const handleTouch = options["handleTouch"] || true;
+    // const handleTouch = options["handleTouch"] || true;
```
Comment (sherwinski) [inline src/js/Drift.js]: Stray comment

File: src/js/Drift.js [hunk 2]
```diff
@@ -53,7 +54,8 @@ export default class Drift {
     const onHide = options["onHide"] || null;
     // Add base styles to the page. See the "Theming"
     // section of README.md for more information.
-    const injectBaseStyles = options["injectBaseStyles"] || true;
+    // const injectBaseStyles = options["injectBaseStyles"] || true;
```
Comment (sherwinski) [inline src/js/Drift.js]: Stray comment

File: src/js/Drift.js [hunk 3]
```diff
@@ -44,7 +44,8 @@ export default class Drift {
     // if `false`, it will switch to inline when `windowWidth <= inlinePane`
     const inlinePane = options["inlinePane"] || 375;
     // If `true`, touch events will trigger the zoom, like mouse events.
-    const handleTouch = options["handleTouch"] || true;
+    // const handleTouch = options["handleTouch"] || true;
+    const handleTouch = "handleTouch" in options ? options["handleTouch"] : true;
```
Comment (sherwinski) [inline src/js/Drift.js]: We can ensure that `handleTouch` gets assigned a truthy value based on what's passed in by making the following change:
`const handleTouch = "handleTouch" in options ? !!options["handleTouch"] : true;`

The same should be done for `injectBaseStyles` on line 58

**Comment by sherwinski** [comment]
Hi @amhofny sorry but it looks like there might have been some confusion - when I asked to have the `dist/*` files removed, I meant to remove your changes on those files. Right now looks like you have removed the files altogether - apologies if what I said previously was confusing. We basically want this PR to only include changes on two files: `src/js/Drift.js` and `test/testDrift.js`.

Aside from that, everything else looks great. As soon as we get the other files sorted out, we'll be ready to merge. Thanks for all your help so far!

**Comment by amhofny** [comment]
Hi @sherwinski, sorry for that i will reset the files to their origin/master state and commit

**Pull request merged**